utils/resolve.js

```diff
@@ -5,6 +5,7 @@ exports.__esModule = true;
 const fs = require('fs');
 const Module = require('module');
 const path = require('path');
+const { getPhysicalFilename } = require('../src/context');
```

Comment:
Wouldn't this import be an issue?  If I understand the package hierarchy correctly, this is being published as a separate package `eslint-module-utils`.  Yet here you're reaching into the other package to grab this function.  This package probably shouldn't pull from the plugin package.  Maybe justifies moving the context to the utils package?

Reply:
that's exactly right, good catch. if it's something both the main and utils package need, it needs to be added to the utils package first as a semver-minor, published, and then the main package can replace its implementation with the one in the utils package along with updating the dep.

If you leave standalone commits on this PR for the "add to utils package" and "update main to use the utils package", then I can pull those in and handle the release dance, and rebase this PR on top of that.

Reply:
If I put the change in to add context to the utils package as a PR to @G-Rath's branch, will that work?  Or I can put it in as an entirely separate pr, whichever you prefer.

Reply:
either works great!

Reply:
https://github.com/import-js/eslint-plugin-import/pull/3049